eZ/Publish/Core/MVC/Symfony/Cache/Http/SymfonyCache/UserContextSubscriber.php

```diff
@@ -1,27 +0,0 @@
-<?php
-
-/**
- * This file is part of the eZ Publish Kernel package.
- *
- * @copyright Copyright (C) eZ Systems AS. All rights reserved.
- * @license For full copyright and license information view LICENSE file distributed with this source code.
- */
-namespace eZ\Publish\Core\MVC\Symfony\Cache\Http\SymfonyCache;
-
-use FOS\HttpCache\SymfonyCache\UserContextSubscriber as BaseUserContextSubscriber;
-use Symfony\Component\HttpFoundation\Request;
-
-/**
- * Extends UserContextSubscriber from FOSHttpCache to include original request.
- *
- * {@inheritdoc}
- */
-class UserContextSubscriber extends BaseUserContextSubscriber
```

Comment:
don't seem to be used anywhere, however if it is we might need to move this to ezplatform-http-cache

Reply:
https://github.com/ezsystems/ezplatform-http-cache/pull/46